packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -61,18 +77,39 @@ function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLEl
     isRequired,
     label: null,
     isQuiet: false,
-    'aria-labelledby': labelProps.id || ariaLabel,
-    validationState: null
+    validationState: null,
+    description: null,
+    errorMessage: null,
+    descriptionProps: null,
+    errorMessageProps: null
```

Comment:
do we need this?
```
'aria-label': null
```
I know picker sets its own, but searchfield should probably null it? or have a name we set?

Reply:
I think we should probably set to null, since it is basically "covered" by the group and we also don't want users to provider their own. Matches the behavior of visual labels here

Reply:
pushed this change; if anyone disagrees, feel free to leave a comment